temporalcli/client.go

```diff
@@ -157,6 +157,14 @@ func clientIdentity() string {
 
 type stringMapHeadersProvider map[string]string
 
+func newStringMapHeadersProvider() stringMapHeadersProvider {
+	rmap := make(map[string]string)
+	if authHeader := os.Getenv(`TEMPORAL_CLI_AUTHORIZATION_TOKEN`); authHeader != "" {
```

Comment:
Can you use `CommandContext.Options.LookupEnv` here? That's out common abstraction for accessing external things like env vars so that tests can set it.

Reply:
Ah, OK. I was wondering why I found so few things.